pkg/loader/util.go

```diff
@@ -259,6 +319,36 @@ func getColsOfTbl(db *gosql.DB, schema, table string) ([]string, error) {
 	return cols, nil
 }
 
+func getOracleColsOfTbl(db *gosql.DB, schema, table string) ([]string, error) {
+	rows, err := db.Query(colsOracleSQL, schema, table)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	defer rows.Close()
+	cols := make([]string, 0, 1)
+	for rows.Next() {
+		var name, virtualColumn string
```

Comment:
Why do we need to select `virtualColumn`? It's never used.

Reply:
yes, it can be deleted.